airqo-platform/AirQo-api
# chore: deploy data-mgt and redis on k8s

# Deploy data-mgt & redis on k8s

**_WHAT DOES THIS PR DO?_**
This PR adds configurations for the deployment of `data-mgt` and `redis` on k8s 
**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
N/A
**_WHAT IS THE LINK TO THE PR BRANCH_**
N/A
**_HOW DO I TEST OUT THIS PR?_**
Locally: `npm run stage-mac` or `npm run stage-pc`. Make sure redis is running 
Docker (stage). `docker-compose up`

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
All (endpoints)[https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=757070984] of data management microservice.
Example endpoint: `http://34.78.78.202:31001/api/v1/data/feeds/transform/recent?channel=689750`
**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A
**_ARE THERE ANY RELATED PRs?_**
N/A
